service/static/js-src/muk.funder.js

```diff
@@ -4004,36 +4004,27 @@ $.extend(true, edges, {
                         if (val.count === 0 && this.hideEmpty) {
                             continue
                         }
-                        if ($.inArray(val.term.toString(), ts.selected) === -1) {
-                            results += '<div class="' + resultClass + '"><a href="#" class="' + valClass + '" data-key="' + edges.escapeHtml(val.term) + '">' + edges.escapeHtml(val.display) + "</a>";
+                        if ($.inArray(val.term.toString(), ts.selected) !== -1) {
+                            results += '<div class="form ' + resultClass + '"><div class="form-fields__item-checkbox ' + filterRemoveClass + '" data-key="' + edges.escapeHtml(val.term) + '"><label><input type="checkbox" checked/> ' + edges.escapeHtml(val.display);
                             if (this.showCount) {
                                 results += ' <span class="' + countClass + '">(' + val.count + ")</span>"
                             }
-                            results += "</div>"
-                        }
-                    }
-                }
-                var filterFrag = "";
-                if (ts.selected.length > 0) {
-                    for (var i = 0; i < ts.selected.length; i++) {
-                        var filt = ts.selected[i];
-                        var def = this._getFilterDef(filt);
-                        if (def) {
-                            filterFrag += '<div class="' + resultClass + '"><strong>' + edges.escapeHtml(def.display);
+                            results += "</label></div></div>"
+                        } else {
+                            results += '<div class="form ' + resultClass + '"><div class="form-fields__item-checkbox ' + valClass + '" data-key="' + edges.escapeHtml(val.term) + '"><label><input type="checkbox" /> ' + edges.escapeHtml(val.display);
                             if (this.showCount) {
-                                filterFrag += " (" + def.count + ")"
+                                results += ' <span class="' + countClass + '">(' + val.count + ")</span>"
                             }
-                            filterFrag += '&nbsp;<a href="#" class="' + filterRemoveClass + '" data-key="' + edges.escapeHtml(def.term) + '">';
-                            filterFrag += '<i class="glyphicon glyphicon-black glyphicon-remove"></i></a>';
-                            filterFrag += "</strong></a></div>"
+                            results += "</label></div></div>"
                         }
                     }
                 }
+
```

Comment:
The above code, from line 4000 to 2021 can most efficiently be coded as (note use of 'self' variable, needed because 'this' has different context inside the $.each( ) callback function.  

           if (ts.terms.length > 0) {
					var self = this;
                    results = '<div class="form ' + resultClass + '">';
					$.each(ts.terms, function(ix,val){
						if (val.count || ! self.hideEmpty) {
								results += '<div class="form-fields__item-checkbox ' + filterRemoveClass + '" data-key="' + edges.escapeHtml(val.term) + '"><label><input type="checkbox"' + (($.inArray(val.term.toString(), ts.selected) !== -1) ? ' checked' : '') + '/> ' + edges.escapeHtml(val.display) + (self.showCount ? ' <span class="' + countClass + '">(' + val.count + ')</span>' : '') + '</label></div>';
						}
					});
					results += '</div>';
                }